README.md

```diff
@@ -1,6 +1,20 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
hey @chibytez , Please fix the merge conflict issue here.

Reply:
have done that ,thanks